ui/app/models/allocation.js

```diff
@@ -30,18 +30,19 @@ export default Model.extend({
     return taskGroups && taskGroups.findBy('name', this.get('taskGroupName'));
   }),
 
-  percentMemory: computed(
-    'taskGroup.reservedMemory',
-    'stats.ResourceUsage.MemoryStats.Cache',
-    function() {
-      const used = this.get('stats.ResourceUsage.MemoryStats.Cache');
-      const total = this.get('taskGroup.reservedMemory');
-      if (!total || !used) {
-        return 0;
-      }
-      return used / total;
+  memoryUsed: computed.readOnly('stats.ResourceUsage.MemoryStats.RSS'),
+  cpuUsed: computed('stats.ResourceUsage.CpuStats.TotalTicks', function() {
+    return Math.floor(this.get('stats.ResourceUsage.CpuStats.TotalTicks'));
+  }),
+
+  percentMemory: computed('taskGroup.reservedMemory', 'memoryUsed', function() {
+    const used = this.get('memoryUsed') / 1024 / 1024;
+    const total = this.get('taskGroup.reservedMemory');
```

Comment:
does the taskGroup only support specifying in one unit? or will you have to do conversions here too?

Reply:
It's always in MiB. In fact it's unitless in the job file: https://github.com/hashicorp/nomad/blob/master/terraform/examples/spark/hdfs.nomad#L74